Sources/Vapor/Utilities/FileIO.swift

```diff
@@ -407,6 +411,8 @@ public struct FileIO: Sendable {
     ///        print("chunk: \(data)")
     ///    }
     ///
+    ///    > Warning: It's the caller's responsibility to close the file handle provided in ``FileChunks`` when finished.
```

Comment:
Still incorrectly indented.
```suggestion
    /// > Warning: It's the caller's responsibility to close the file handle provided in ``FileChunks`` when finished.
```